hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/auth/Roles.java

```diff
@@ -1,8 +1,9 @@
 package pl.allegro.tech.hermes.management.api.auth;
 
 public final class Roles {
-    public static final String SUBSCRIPTION_OWNER = "subscriptionOwner";
-    public static final String TOPIC_OWNER = "topicOwner";
-    public static final String ADMIN = "admin";
     public static final String ANY = "any";
+    public static final String ADMIN = "admin";
+    public static final String RESOURCE_WITHOUT_OWNER = "resourceWithoutOwner";
```

Comment:
how about `RESOURCE_WITHOUT_OWNERSHIP_OWNER` ;) as it describes the user role that has rights to specific resource, not the resource itself?